Source/WebCore/page/Navigation.h

```diff
@@ -186,6 +186,7 @@ class Navigation final : public RefCounted<Navigation>, public EventTarget, publ
     };
     Ref<AbortHandler> registerAbortHandler();
 
+    NavigateEvent* ongoingNavigateEvent() { return m_ongoingNavigateEvent.get(); }
```

Comment:
Can we suffix this function name with "Concurrently" to emphasize the fact this function runs concurrently to the main thread?

Reply:
This is just a generic getter though, we wouldn't write it any differently if it wasn't called from a background thread and it can totally get called from the main thread, for non-GC purposes. Therefore, I don't quite understand the purpose.

Reply:
In that case, can we add a comment saying this can be called in non-main thread? Otherwise, I'm afraid someone will try to introduce Ref/RefPtr/CheckedRef/CheckedPtr or other non-thread-safe constructs there.